src/routes/about/+page.svelte

```diff
@@ -38,8 +38,13 @@
 <h2>Constitution</h2>
 <p>
 	As required by the University of Michigan and the SOAR (Student Organization and Recognition)
-	Initiative, our club constitution can be found on <a
-		href="https://maizepages.umich.edu/organization/kendoclub"
-		>Maize pages.
-	</a>
+	Initiative, our club constitution can be found on
+	<a href="https://maizepages.umich.edu/organization/kendoclub">Maize pages</a>.
 </p>
+
+<style>
+	.motto {
+		font-size: 1.5em;
+		letter-spacing: 0.1em;
```

Comment:
I did a brief search for the moto on Google and it seemed like everyone else let them run up against each other. Are you sure the spacing improves reliability it seems like Japanese websites don't add spacing.

Reply:
I think it does. I can support it with these examples: 
![image](https://github.com/kendoclub-at-umich/website/assets/75005450/cbb0ce39-4656-40ec-ac0f-8ea7ef21ba4d)
![image](https://github.com/kendoclub-at-umich/website/assets/75005450/08e5c4c7-1a8f-42ac-87a2-62b975d7b505)
![image](https://github.com/kendoclub-at-umich/website/assets/75005450/62256f25-d73d-44dd-8578-d2580866371f)

Reply:
your second example seem to be using a different set of glyphs that the moto. Look how simple they are. In most of the examples it looks like the extra space around the dash looking glyph comes from the shape of the nearby glyphs, look at how they don't have a mark in the middle but they have marks sticking out on the top or bottom. But look at how close it gets at the end of the dash on the first one you circled. That's even closer than ours without the spacing.

It looks like ours has a spacing of about 1 stroke width on the main branch and 2 stroke widths on this branch. I think both screenshots have a spacing of slightly less than one stroke width when you look at the places where the glyphs naturally comes together. Look the the green text next to the old lady on the bottom left of the first screenshot, or the second dash looking glyph next to the man with the yellow hat.

I think we should trust Google to know how to display Japanese better than you. Probably you are used to beginner friendly Japanese which has bigger spacing to help you learn.

Reply:
But yes, I am used to more spacing as a beginner, mostly because I'm used to using the simpler character sets (called Hiragana and Katakana) that have more spacing between each other naturally because they have alot less complexity in shape. The complex character set you're referring to is "Kanji", which is the Chinese characters used for words, and yes, the Kanji spacing is quite close on the examples I showed you. 
I still feel that our font puts glyphs a bit closer together than other examples I've seen. I think slightly more spacing is nice so that they don't feel cramped, especially since the motto is so short. And also when the dash glyph and the next glyph aren't touching, it's more aesthetically pleasing, even at 0.05em instead of 0.1em. 
However, if you think it's important to trust the font, I will trust your judgement and go back to no letter spacing.

Reply:
My phone looks different than your screenshots can you try it on Android and see if you can figure out what's going on?

Reply:
...Mine does too. It's subtle, but there might be a difference in font? The diagonal part of the first part of the glyph after the dash glyph seems longer on my computer than it is on my phone. Trying to figure it out
![Screenshot_20240522-162557](https://github.com/kendoclub-at-umich/website/assets/75005450/4669ca09-3e74-4fc8-ac82-a4de0449f615)

Reply:
Alright, I removed letter spacing for now, as we have feedback it looks better on Android without, and on desktop it looks well enough. Resolved in [5856964](https://github.com/kendoclub-at-umich/website/pull/45/commits/5856964fbce9fa924724e29bd154ed76fe372731)